### PR TITLE
add a flag for FIPS enabled/disabled at /api/v2/config

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

**Comment by wenottingham** [comment]
... why?  If this is needed for sosreport-y sort of things, it would already be captured there.

**Comment by ryanpetrello** [comment]
image

File: awx/ui/client/src/about/about.controller.js [hunk 1]
```diff
@@ -6,14 +6,18 @@ export default ['$rootScope', '$scope', '$location', 'ConfigService', 'lastPath'
             $scope.version = config.version.split('-')[0];
             $scope.ansible_version = config.ansible_version;
             $scope.subscription = config.license_info.subscription_name;
-            $scope.speechBubble = createSpeechBubble($rootScope.BRAND_NAME, config.version);
+            $scope.speechBubble = createSpeechBubble(
+                $rootScope.BRAND_NAME,
+                config.version,
+                config.fips ? ' (FIPS Mode)' : ''
+            );
```
Comment (wenottingham) [inline awx/ui/client/src/about/about.controller.js:13]: Highly-meh-to-slightly-negative about putting this in the UI.

**Comment by matburt** [comment]
One reason I'm not super big on this is because it doesn't really give information about *all* cluster nodes (just the one that you are on) and it won't say anything about future execution nodes (and especially not isolated nodes). I think it might end up leading to more confusion than help

**Comment by ryanpetrello** [comment]
These are good points.  I agree that putting this in the sosreport is probably more useful 👍.

**Comment by rooftopcellist** [comment]
If FIPS is enabled, it will show up in the sosreport here: `proc/sys/crypto/fips_enabled`.  

```
sosreport
cat /var/tmp/<sosreport>/proc/sys/crypto/fips_enabled
```

**Pull request closed without merging**